picky-server/src/db/file.rs

```diff
@@ -255,6 +273,28 @@ impl PickyStorage for FileStorage {
         .boxed()
     }
 
+    fn increase_issued_authenticode_timestamps_counter(&self) -> BoxFuture<'_, Result<(), StorageError>> {
+        let name = format!("{}{}", "timestamp_counter_store", TXT_EXT).replace(" ", "_");
+
+        async move {
+            let mut content = 0_u32.to_le_bytes().to_vec();
```

Comment:
Assuming you fix the `get` function, this should become
```suggestion
            let mut content = [0; 4];
```